src/test/java/org/broadinstitute/hellbender/tools/walkers/mutect/CountFalsePositivesTest.java

```diff
@@ -24,20 +26,18 @@
      */
     @Test
     public void testEvaluateMutect2() throws Exception {
-        final String dreamDir =  publicTestDir + "org/broadinstitute/hellbender/tools/mutect/dream/vcfs/";
-        List<String> vcfList = Arrays.asList(dreamDir + "sample_1.vcf", dreamDir + "sample_2.vcf", dreamDir + "sample_3.vcf", dreamDir + "sample_4.vcf");
-        final File vcfListFile = createTempFile("vcfList", ".txt");
+        final String dreamDir =  publicTestDir + "org/broadinstitute/hellbender/tools/mutect/dream/";
         final File output = createTempFile("output", ".txt");
-        Path file = Paths.get(vcfListFile.getAbsolutePath());
-        Files.write(file, vcfList, Charset.forName("UTF-8"));
         final String[] args = {
-                "-V", vcfListFile.toString(),
+                "-V", dreamDir + "vcfs/dream3-chr20.vcf",
+                "-R", b37_reference_20_21,
+                "-L", dreamDir + "dream-chr20.interval_list",
                 "-O", output.toString()
         };
         runCommandLine(args);
-        Stream<String> outputLines = Files.lines(Paths.get(output.getAbsolutePath()));
-        Assert.assertEquals(outputLines.count(), 5L);
-
     }
 
+    // TODO: write a robust test using TableReader
```

Comment:
You should implement this: write a `TableReader` for `FalsePositiveRecord`, and then write `assert`s for the number of SNPs, the number of indels, and the rates per interval, which you can obtain manually from your short test vcfs.